event_routing_backends/processors/xapi/transformer_processor.py

```diff
@@ -34,7 +34,7 @@ def transform_event(self, event):
             event (dict):   Event to be transformed.
 
         Returns:
-            dict:           transformed event
+            ANY:            transformed event
```

Comment:
This was always true (e.g. xAPI returns a tincat Statement), but the docstring hadn't been updated.